### PR TITLE
Add sendPreviousResourceOnDelete to fhir-notification-configs

### PR DESCRIPTION
**Pull request by JosieLi-Google**
```release-note: enhancement
healthcare: added `send_previous_resource_on_delete`
```

### CODE REVIEW

File: mmv1/products/healthcare/FhirStore.yaml
Comment (roaks3) [inline mmv1/products/healthcare/FhirStore.yaml:293]: Typo: should be "previous"

File: mmv1/products/healthcare/FhirStore.yaml
Comment (roaks3) [inline mmv1/products/healthcare/FhirStore.yaml:297]: Same as above with "previous"

**Review by roaks3 — CHANGES_REQUESTED** [review]
I found some typos, but otherwise seems fine after a first pass.

I'll run tests now, but also make sure you've gone through https://g3doc.corp.google.com/company/teams/terraform/users/resource-coverage/contribution-guide/index.md?cl=head to make sure you're in the appropriate groups to have the tests run without needing approval.

Lastly, it looks like you may need to update the PR description to get the release note formatted properly.

**Comment by roaks3** [comment]
Build error is:
```
/usr/local/lib/ruby/3.1.0/psych.rb:455:in `parse': (<unknown>): found a tab character where an indentation space is expected while scanning a block scalar at line 284 column 24 (Psych::SyntaxError)
```

I'm assuming there is a tab somewhere in your yaml that is making something unhappy.

**Comment by roaks3** [comment]
Quick note while waiting for the tests: your `PRODUCT` in the release note would be `healthcare`

**Comment by JosieLi-Google** [comment]
Hi @roaks3 , to resolve the Diff report, should I simple `git rebase`?

**Comment by roaks3** [comment]
@JosieLi-Google it looks like something strange has happened since your [first commit](https://github.com/GoogleCloudPlatform/magic-modules/pull/8361/commits/fc007a168b8ad73d9d3b2f06f9441a8a40bf661d) (maybe the merges?). The first commit appears to add a field, but if I look at the file changes for the PR currently, I'm only seeing changes to the description (and some other small tweaks).

I would suggest cleaning up the branch with something like this:
```
git rebase -i fc007a168b8ad73d9d3b2f06f9441a8a40bf661d
<remove merge commit lines from the file before saving>
```

Hopefully that cleans up whatever happened with the merges, and you can try:
```
git fetch origin main
git rebase origin/main
```

Which would catch up the branch, with your handful of commits on top.

If that doesn't work, you of course have other options, like `git reset` + `git cherry-pick`, or just starting fresh and copy-and-pasting.

**Comment by JosieLi-Google** [comment]
> @JosieLi-Google it looks like something strange has happened since your [first commit](https://github.com/GoogleCloudPlatform/magic-modules/pull/8361/commits/fc007a168b8ad73d9d3b2f06f9441a8a40bf661d) (maybe the merges?). The first commit appears to add a field, but if I look at the file changes for the PR currently, I'm only seeing changes to the description (and some other small tweaks).
> 
> I would suggest cleaning up the branch with something like this:
> 
> ```
> git rebase -i fc007a168b8ad73d9d3b2f06f9441a8a40bf661d
> <remove merge commit lines from the file before saving>
> ```
> 
> Hopefully that cleans up whatever happened with the merges, and you can try:
> 
> ```
> git fetch origin main
> git rebase origin/main
> ```
> 
> Which would catch up the branch, with your handful of commits on top.
> 
> If that doesn't work, you of course have other options, like `git reset` + `git cherry-pick`, or just starting fresh and copy-and-pasting.

Hi @roak3, I think my changes are already submitted in https://github.com/GoogleCloudPlatform/magic-modules/pull/8507. I will double check if there is anything still missing, if not I will close the PR. Let me know if there is any concern. Thank you!

**Comment by roaks3** [comment]
Ah ok, that explains the strange diff after the merges. Thanks!

**Comment by roaks3** [comment]
@JosieLi-Google gentle ping on this. Since it has been sitting a while, we may just want to close this PR and come back if there is anything missing.

**Pull request closed without merging**
